README.md

```diff
@@ -44,21 +44,24 @@ defmodule MyApp.Factory do
   def factory(:user, _attrs) do
     %User{
       name: "Jane Smith",
-      email: sequence(:email, &"email-#{&1}@example.com"),
```

Comment:
For some reason I thought these weren't allowed in Elixir. I'll change it back :)